front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -105,8 +105,8 @@ const useStdcm = ({
           setStdcmTrainResult(stdcmTrain);
           dispatch(updateSelectedTrainId(STDCM_TRAIN_ID));
         } else {
-          // When the back-end send back result with status 'path_not_found' we consider that the result is empty
-          setIsStdcmResultsEmpty(response.status === 'path_not_found');
+          // When the back-end send back result with status 'conflicts' we consider that the result is empty
+          setIsStdcmResultsEmpty(response.status === 'conflicts');
```

Comment:
This might conflict with @achrafmohye PR : https://github.com/OpenRailAssociation/osrd/pull/9419

Reply:
Ok, I’ll wait for this PR to be merged before.

Reply:
I rebased, so there's no more conflict.